spirv_msl.cpp

```diff
@@ -12982,8 +13046,8 @@ string CompilerMSL::to_name(uint32_t id, bool allow_alias) const
 	return Compiler::to_name(id, allow_alias);
 }
 
-// Returns a name that combines the name of the struct with the name of the member, except for Builtins
-string CompilerMSL::to_qualified_member_name(const SPIRType &type, uint32_t index)
+// Appends the name of the member to the variable qualifier string, except for Builtins.
+string CompilerMSL::append_member_name(const string qualifier, const SPIRType &type, uint32_t index)
```

Comment:
`const string` should probably be `const string &` here.

Reply:
Fixed.